openlibrary/plugins/openlibrary/js/dialog.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Wires up dialog close buttons
+ */
+export default function dialogs() {
+    // This will close the dialog in the current page.
+    $('.floaterShut').attr('href', 'javascript:;').on('click', () => $.fn.colorbox.close());
```

Comment:
`floaterShut` is a weird name; Let's rename it to `dialog--close` and `dialog--close-parent`.

(There are currently 3 ways these things are referred to throughout the code: `floater`, `popup`, `dialog`, `colorbox`; we should consolidate to one.)

Reply:
Agreed. Do you mind if i do this in a follow up? It will be easier to  change all the instances then.

Do I need to worry about cached HTML? If so, I can do a combined selector to account for both e.g. ``` .floaterShut,.dialog--close```

Reply:
I would rather have it error than have to do that :P Doing in a follow up sounds good :+1:

Reply:
Although we should look into automatically adding `?v=COMMIT_HASH` to the webpack dynamic imports; that would solve this issue.